pages/animestatic/[animedetailstatic].js

```diff
@@ -53,9 +44,8 @@ export async function getStaticPaths(){
 export async function getStaticProps(context){
```

Comment:
cant we call the apiCall and in useeffect and render the data then, and have a loader before the data is fetched? @deepcode3